### PR TITLE
feat(deb): add common dist

### PR DESCRIPTION
**Pull request by DmitriyLewen**
## Description
We use same deb packages for every Ubuntu/Debian version.
e.g. you can install Trivy on Ubuntu 22.04 using `bookworm` dir.
```
root@6ea9f26f4810:/# cat /etc/os-release 
PRETTY_NAME="Ubuntu 22.04.3 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.3 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
...

root@6ea9f26f4810:/# cat /etc/apt/sources.list.d/trivy.list
deb [signed-by=/usr/share/keyrings/trivy.gpg] https://aquasecurity.github.io/trivy-repo/deb bookworm main

root@6ea9f26f4810:/# apt-get install trivy
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
...
Unpacking trivy (0.50.2) ...
Setting up trivy (0.50.2) ...
```

This way we can use a `common` folder for all Ubuntu/Debian releases.

### CODE REVIEW

**Comment by knqyf263** [comment]
Great. Is there any conventional name? Do other projects use `common`?

**Comment by knqyf263** [comment]
>K8s uses root dir - https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-using-native-package-management

It looks good. Wdyt?

**Comment by DmitriyLewen** [comment]
IIUC we can't do that using `reprepro`.

see https://manpages.debian.org/jessie/reprepro/reprepro.1.en.html#conf/distributions:

> Codename
    This required field is the unique identifier of a distribution and used as directory name within dists/ It is also copied into the Release files.

- `dists` dir is always using.
- `Codename` can't be empty.

**Comment by knqyf263** [comment]
OK. Both `universal` and `generic` sound good to me.

**Comment by DmitriyLewen** [comment]
Renamed to `generic` in https://github.com/aquasecurity/trivy-repo/pull/37/commits/f638bdc732a337a545895d9c84a98d652d459854

File: README.md [hunk 1]
```diff
@@ -4,15 +4,13 @@ deb/rpm repository for Trivy
 ## Debian/Ubuntu
 
 ```
-$ sudo apt-get install apt-transport-https
-$ wget -qO - https://aquasecurity.github.io/trivy-repo/deb/public.key | sudo apt-key add -
-$ echo deb https://aquasecurity.github.io/trivy-repo/deb [CODE_NAME] main | sudo tee -a /etc/apt/sources.list
+$ apt-get install wget apt-transport-https gnupg
```
Comment (knqyf263) [inline README.md:7]: It's better to keep `sudo` for non-root users.

```suggestion
$ sudo apt-get install wget apt-transport-https gnupg
```

**Comment by DmitriyLewen** [comment]
Created PR to update link in Trivy docs - https://github.com/aquasecurity/trivy/pull/6606

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**